cirq-core/cirq/ops/pauli_string.py

```diff
@@ -1103,17 +1095,10 @@ def _validate_qubit_mapping(
 def _try_interpret_as_pauli_string(op: Any):
     """Return a reprepresentation of an operation as a pauli string, if it is possible."""
     if isinstance(op, gate_operation.GateOperation):
```

Comment:
I think the check for GateOperation can be relaxed to Operation.

Reply:
Also, minor nit, can you invert the `if` blocks so that they return `None` if the condition is false, to avoid the extra indentation levels.